<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerDialog</class>
 <widget class="QDialog" name="PlayerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#pushButton_play{
border-image: url(:/prefix1/play.png);
}
#pushButton_play:pressed{
border-image: url(:/prefix1/play1.png);
}
#pushButton_pause{
border-image: url(:/prefix1/pause.png);
}
#pushButton_pause:pressed{
border-image: url(:/prefix1/pause1.png);
}
#pushButton_stop{
border-image: url(:/prefix1/stop.png);
}
#pushButton_stop1:pressed{
border-image: url(:/prefix1/stop1.png);
}
#pushButton_cap{
border-image: url(:/prefix1/snapshot.png);
}
#pushButton_cap:pressed{
border-image: url(:/prefix1/snapshot1.png);
}
#pushButton_open{
border-image: url(:/prefix1/opendir.png);
}
#pushButton_open:pressed{
border-image: url(:/prefix1/opendir1.png);
}
#pushButton_forward{
border-image: url(:/prefix1/forward.png);
}
#pushButton_forward:pressed{
border-image: url(:/prefix1/forward1.png);
}
#pushButton_close{
border-image: url(:/prefix1/close2.png);
}

#pushButton_rewind{
border-image: url(:/prefix1/rewind.png);
}
#pushButton_rewind:pressed{
border-image: url(:/prefix1/rewind1.png);
}
#PlayerDialog{
border-image: url(:/prefix1/playerborder.png);
}
#widget,#graphicsView{
background:#000000;
}
#label_speed,#label_name
{
color:#FFFFFF;
}</string>
  </property>
  <widget class="QPushButton" name="pushButton_pause">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>580</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_stop">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>580</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_play">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>570</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlide_speedr">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>540</y>
     <width>591</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_speed">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>540</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_open">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>580</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>590</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cap">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>580</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_forward">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>580</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_rewind">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>580</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>981</width>
     <height>495</height>
    </rect>
   </property>
   <widget class="QMySelfVideo" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>1</y>
      <width>617</width>
      <height>480</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>10</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QMySelfVideo</class>
   <extends>QGraphicsView</extends>
   <header location="global">qmyselfvideo.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
